restlight-integration-test/springmvc-test/src/main/java/io/esastack/restlight/integration/cases/controller/FilterController.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 OPPO ESA Stack Project
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.esastack.restlight.integration.cases.controller;
+
+import io.esastack.restlight.integration.entity.UserData;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @author chenglu
+ */
+@RestController
+@RequestMapping("/filter/")
+public class FilterController {
+
+    @GetMapping("get")
+    public UserData filter(@RequestHeader String name) {
+        return UserData.Builder.aRestResult()
```

Comment:
Does aRestResult() have any special meaning? Logically, it is not necessary to call this method to build UserData.